gen3/docs/gen3-resources/operator-guide/helm/index.md

```diff
@@ -0,0 +1,9 @@
+# Helm to Configure and Deploy Gen3
+
+https://docs.gen3.org/docs/Deployment/Helm%20Overview
+
+https://docs.gen3.org/docs/Deployment/Deployment%20Architecture/
+
+* Note: There’s some sensible defaults that ship with helm, and if you don’t provide overrides it will fall back to the defaults.
+* This is something we need to document more as Gen3 is HIGHLY configurable. Probably need a whole section, or a small book about this
```

Comment:
We should probably not include these comments on the website, but rather add them as hidden comments and label them with "TODO" so we can populate this section at a later time.